python/ql/lib/change-notes/2024-06-24-Data-extensions-supported-by-queries-and-documented.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* A number of Python queries now support sinks defined vi data extensions. The format of data extensions for Python has been documented.
```

Comment:
```suggestion
* A number of Python queries now support sinks defined using data extensions. The format of data extensions for Python has been documented.
```
Typo, but also maybe a more plain word here will suffice.

Reply:
Oops, I just saw the approval and not this comment..